.circleci/config.yml

```diff
@@ -130,15 +130,40 @@ jobs:
       - attach_workspace:
           at: ~/project
       - run:
-          name: Login for Docker
-          command: echo $DOCKER_TOKEN | docker login -u "$DOCKER_USER" --password-stdin
+          name: Log into Github Docker Registry
+          command: |
+              echo $QLIK_DOCKER_DEV_PASSWORD | docker login -u $QLIK_DOCKER_DEV_USERNAME --password-stdin $QLIK_DOCKER_DEV_REGISTRY
```

Comment:
change back to using DOCKER_TOKEN and DOCKER_USER
or add QLIK_DOCKER_DEV_PASSWORD and QLIK_DOCKER_DEV_USERNAME to circle ci
(looks like QLIK_DOCKER_DEV_REGISTRY already exists)

Reply:
Because of the public image, we have removed this step.